spring-thrift-starter/src/main/java/ru/trylogic/spring/boot/thrift/beans/RequestIdLogger.java

```diff
@@ -16,7 +16,7 @@
   private static final String DEFAULT_MDC_KEY = "request_id";
   private static final String X_REQUEST_ID = "x-request-id";
 
-  public String getMDCKey() {
+  public static String getMDCKey() {
```

Comment:
I've made it static just to reuse in a `RequestIdAwareThreadPoolExecutor` (can't inject since the thread pool is not a bean). Is it better to extract this constant into separate dictionary class?